### ISSUE TITLE
Auth adapters

### ISSUE THREAD

**Issue by mruz** [mruz/base-app#23]
It would be nice to see other adapters than model: collection, file.

**Comment by mruz** [comment]
I't possible in [ice/base](https://github.com/ice/base).

**Comment by NeoVance** [comment]
I would like to help. Can you provide a more detailed explanation of this feature?

**Comment by mruz** [comment]
We schould rewrite the [/app/common/library/Auth.php](https://github.com/mruz/base-app/blob/master/app/common/library/Auth.php) to be able work with collections (mongoDB). It could use [MVC: Entity Management ](https://github.com/phalcon/cphalcon/pull/2917) instead of using only models.

**Comment by NeoVance** [comment]
Looks like that change was never merged, so we would have to create this functionality as a new layer over phalcon.

**Comment by mruz** [comment]
Yes, the changes was never merged :sad: so we should create own layer with adapters like model, collection or [file](https://github.com/kohana/auth/blob/3.3/master/classes/Kohana/Auth/File.php).
